packages/@react-spectrum/layout/docs/Flex.mdx

```diff
@@ -136,7 +136,7 @@ an `alignItems` value to account for this.
 ### Justification
 
 The `justifyContent` prop can be used to align items along the main axis. When `direction` is `"column"`, this refers
-to vertical alignment, and when `direction` is `"column"` it refers to horizontal alignment.
+to horizontal alignment, and when `direction` is `"row"` it refers to vertical alignment.
```

Comment:
```suggestion
to vertical alignment, and when `direction` is `"row"` it refers to horizontal alignment.
```

Reply:
Of course, justify-content aligns items on the main axis!
I signed the CLA, and hopefully the docs and tests will all succeed now as well.